nbconvert/templates/html/basic.tpl

```diff
@@ -209,3 +209,31 @@ var element = $('#{{ div_id }}');
 </script>
 </div>
 {%- endblock -%}
+
+{%- block data_widget_state scoped %}
+{% set div_id = uuid4() %}
+{% set content = output.data['application/vnd.jupyter.widget-state+json'] %}
+<div id="{{ div_id }}"></div>
+<div class="output_subarea output_widget_state {{extra_class}}">
+<script type="tapplication/vnd.jupyter.widget-state+json">
+var element = $('#{{ div_id }}');
+{
+"model_id": "{{content.model_id}}"
+}
+</script>
```

Comment:
in response to the bit regarding the notebook metadata, thank you for the link. I'm going to try to apply the current version of nbconvert to that notebook, but am running into issues that have to do with my own system.

About the second, is there anything you would like to have done with the contents of either `version_major` or `version_minor`or is the current "dumping it all in between two curly braces" approach going to cover that as well? My intuition is that it does cover that, but I could be wrong.